src/diffusers/schedulers/scheduling_ddpm.py

```diff
@@ -204,6 +204,7 @@ def _get_variance(self, t, predicted_variance=None, variance_type=None):
         # for rl-diffuser https://arxiv.org/abs/2205.09991
         elif variance_type == "fixed_small_log":
             variance = torch.log(torch.clamp(variance, min=1e-20))
+            variance = torch.exp(0.5 * variance)
```

Comment:
Uff that's a pretty big change - do we need to do this? Is this not affecting any of our DDPM checkpoints/tests?

Reply:
@patrickvonplaten I added this `variance_type` condition specifically for the RL implementation. Double checked that nothing else in the code base uses it.